algorithm/partition_copy.md

```diff
@@ -0,0 +1,29 @@
+# unique_copy
```

Comment:
The title is wrong.
apart from that everything looks good to me 👍🏽

Reply:
Thanks for your review, I have now fixed the title.
Happy hacktoberfest!